skore/src/skore/sklearn/_plot/metrics/roc_curve.py

```diff
@@ -843,6 +843,7 @@ def _compute_data_for_display(
         *,
         report_type: ReportType,
         estimators: Sequence[BaseEstimator],
+        estimator_names: list[str],
```

Comment:
why do you need this new parameter in this function? I don't see where it's used.